### PR TITLE
Fixes the arena shuttle by demodularising it

### PR DESCRIPTION
**Pull request by ChesterTheCheesy**
# General Documentation

### Intent of your Pull Request
The "arena" part of emergency arena didn't spawn, so instead i had to make it load separitevely.

### Why is this change good for the game?
~BLOOD FOR THE BLOOD GOD~
Fixes the most fun shuttle you can have. Improves RP

### Briefly describe your PR and the impacts of it, in layman's terms. 
Arena shuttle now works. I also made the shuttle itself be an actual shuttle, it has few shuttle seats, proper engines, shuttle console and Gumdrop, the friendly slaughter demon ship navigator! Arena itself now is on it's own separate zlevel.
Oh and yeah, you don't need the broken achievement to buy the shuttle anymore, now it actually checks for a dead non-admin spawned bubblegum.

### What should players be aware of when it comes to the changes your PR is implementing?
RIP AND TEAR!!

### What general grouping does this PR fall under? 
Emergency Arena shuttle fix

# Changelog
:cl:  
bugfix: fixes arena shuttle not spawning correctly  
tweak: arena shuttle now has engines, a shuttle console, seats and a slaughter demon flight attendant
/:cl:

### CODE REVIEW

**Comment by patpol4** [comment]
Is this tested?

**Comment by ChesterTheCheesy** [comment]
> 
> 
> Is this tested?

It worked when i compiled it

**Review by patpol4 — APPROVED** [review]
Moved some floors around to make it look less bland, added mood lighting, moved the chairs, moved the walls, moved the spawnpoints.

The map part of this is fine but for the person reviewing this keep in mind im not sure if anything about this will work or is balanced because i didnt test it.

**Comment by ChesterTheCheesy** [comment]
holy fuck wej the off-center engines are making my OCD go nuclear

**Comment by patpol4** [comment]
> holy fuck wej the off-center engines are making my OCD go nuclear

oh god how did i not see this

**Comment by patpol4** [comment]
there

**Review by patpol4 — CHANGES_REQUESTED** [review]
Actually no hold on, still have to fix one thing

**Review by patpol4 — APPROVED** [review]
Map is fine now, the checks are being fucky wucky but im pretty sure its unrelated to the map.

**Review by redmoogle — APPROVED** [review]
